tests/tests_app/core/test_constants.py

```diff
@@ -1,9 +1,19 @@
 import os
 from unittest import mock
 
-from lightning_app.core.constants import get_lightning_cloud_url
+from lightning_app.core.constants import get_cluster_driver, get_lightning_cloud_url
 
 
 @mock.patch.dict(os.environ, {"LIGHTNING_CLOUD_URL": "https://beta.lightning.ai"})
 def test_defaults():
     assert get_lightning_cloud_url() == "https://beta.lightning.ai"
+
+
+def test_cluster_drive(monkeypatch):
```

Comment:
The code path for ValueError is not covered

Reply:
Done !